packages/web-components/src/dropdown/dropdown.stories.ts

```diff
@@ -36,7 +36,7 @@ const dropdownTemplate = html<StoryArgs<FluentDropdown>>`
 
 const storyTemplate = html<StoryArgs<FluentDropdown>>`
   <fluent-field ?disabled="${story => story.disabled}">
-    <label slot="label"> <fluent-label required>${story => story.placeholder}</fluent-label></label>
+    <label slot="label"> <fluent-label>${story => story.placeholder}</fluent-label></label>
```

Comment:
The `fluent-field` component takes care of the required state now so this can be reduced:

```suggestion
    <label slot="label">${story => story.placeholder}</label>
```